src/openrct2/core/String.cpp

```diff
@@ -343,65 +324,60 @@ namespace String
         return buffer;
     }
 
-    utf8* Format(const utf8* format, ...)
+    std::string StdFormat(const utf8* format, ...)
     {
         va_list args;
         va_start(args, format);
-        utf8* result = Format_VA(format, args);
+        std::string result = Format_VA(format, args);
         va_end(args);
         return result;
     }
 
-    utf8* Format_VA(const utf8* format, va_list args)
+    std::string Format_VA(const utf8* format, va_list args)
```

Comment:
This previously returned utf8*, so use u8string instead of std::string.

Reply:
Replaced Format_VA std::string return type with u8string